### PR TITLE
Mail improvements along with the updates-testing digest

### PR DESCRIPTION
**Pull request by lmacken**
:mailbox_with_mail:

### CODE REVIEW

File: bodhi/masher.py [hunk 1]
```diff
@@ -613,22 +614,96 @@ def send_notifications(self):
...
+            updlist.sort()
+            for pkg in updlist:
+                maildata += u'    %s\n' % pkg
+            maildata += u'\nDetails about builds:\n\n'
+            for nvr in updlist:
+                maildata += u"\n" + self.testing_digest[prefix][nvr]
+
+            release = self.db.query(Release).filter_by(long_name=prefix).one()
+            mail.send_mail(config.get('bodhi_email'),
+                           config.get('%s_test_announce_list' %
```
Comment (ralphbean) [inline bodhi/masher.py]: Any fallback if this '%s_test_announce_list' address isn't defined in the config?  Maybe have it mail admin@fp.o or something?

File: bodhi/tests/test_masher.py [hunk 2]
```diff
@@ -327,6 +328,11 @@ def test_testing_digest(self, *args):
 
 """ % time.strftime('%Y'))
 
+        mail.assert_called_with(config.get('bodhi_email'), config.get('fedora_test_announce_list'), mock.ANY)
+        assert len(mail.mock_calls) == 2, len(mail.mock_calls)
+        body = mail.mock_calls[1][1][2]
```
Comment (ralphbean) [inline bodhi/tests/test_masher.py]: Wow.  :1234:

**Comment by ralphbean** [comment]
:+1: as is.  Left one question.

File: bodhi/masher.py (hunk 1 above)
Comment (lmacken) [inline bodhi/masher.py]: I'm not sure there's a sane fallback here, so I added a warning instead.

File: bodhi/tests/test_masher.py (hunk 2 above)
Comment (lmacken) [inline bodhi/tests/test_masher.py]: Yep. Super fragile. :hankey:

**Comment by lmacken** [comment]
Cool, thanks for the review @ralphbean

**Pull request merged**